src/en/client.md

```diff
@@ -85,8 +82,33 @@ tar -xzf juju-yymmdd-hhmmss.tar.gz
     This command will extract the contents of the archive and overwrite any
     existing files in the Juju directory. Make sure that this is what you want.
 
+## Client upgrades
+
+The client software is managed by the operating system's package management
+system. On Ubuntu this is traditionally APT:
+
+```bash
+sudo apt update
+sudo apt install juju
+```
+
+If you have installed Juju via the snap package:
+
+```bash
+sudo snap refresh juju
```

Comment:
my only comment is that it may be worth mentioning snaps update automatically.

Reply:
Good idea.